pkg/handler/repositories.go

```diff
@@ -297,6 +303,9 @@ func (rh *RepositoryHandler) update(c echo.Context, fillDefaults bool) error {
 	if err := c.Bind(&repoParams); err != nil {
 		return ce.NewErrorResponse(http.StatusBadRequest, "Error binding parameters", err.Error())
 	}
+	if err := CheckSnapshotForRepo(c, repoParams); err != nil {
+		return err
+	}
```

Comment:
i stuck it here because the user can turn on snapshotting here.  This will eventually trigger a snapshot (i'm working on that now on another pr).  It also would trigger a snapshot via the cron job (later work)